test/validation.html

```diff
@@ -337,5 +360,118 @@
 
       });
     });
+
+    describe('invalid cannot be set to false', () => {
+      let element;
+
+      function nextRender(element) {
+        return new Promise(resolve => {
+          Polymer.RenderStatus.afterNextRender(element, resolve);
+        });
+      }
+
+      describe('text-field invalid', () => {
+        beforeEach(async() => {
+          element = fixtureSync('<vaadin-text-field></vaadin-text-field>');
```

Comment:
Let's have a separate file per component – `text-field-validation.html`, `number-field-validation.html`, and so on – with the following structure:

```js
describe('text-field validation', () => {
  describe('initial', () => {
    // add tests related to the initial validation here
  });

  describe('basic', () => {
    // add the validated event tests here
  });

  describe('invalid cannot be set to false', () => {
    // add tests related to shouldSetInvalid here
  });
});
```